wolfcrypt/src/asn.c

```diff
@@ -39168,47 +39171,64 @@ static int ParseCRL_Extensions(DecodedCRL* dcrl, const byte* buf, word32 idx,
             /* Length of extension data. */
             int length = (int)dataASN[CERTEXTASN_IDX_VAL].length;
 
-            if (oid == AUTH_KEY_OID) {
-            #ifndef NO_SKID
-                /* Parse Authority Key Id extension.
-                 * idx is at start of OCTET_STRING data. */
-                ret = ParseCRL_AuthKeyIdExt(buf + idx, length, dcrl);
-                if (ret != 0) {
-                    WOLFSSL_MSG("\tcouldn't parse AuthKeyId extension");
-                }
-            #endif
+            /* Check for duplicate extension */
```

Comment:
Please comment with the RFC for the reason this is enforced. Also should it be guarded by WOLFSSL_NO_ASN_STRICT? i.e does it differ in behavior for OpenSSL compatibility uses and with wolfCLU?

Reply:
More what I'm concerned about is that our wolfCLU app after these changes can not parse the bad example CRL with duplicate extensions (which yes our library should not act on by default...) but OpenSSL command line utility can parse it.